.nvmrc

```
@@ -1 +1 @@
-v18.17.0
+v22.18.0
```

suggestion: only set the major version and not the complete version to automatically get patch updates.


```suggestion
22
```

```
❯ echo 22 > .nvmrc

❯ nvm install
Now using Node v22.18.0 (npm 10.9.3) ~/.local/share/nvm/v22.18.0/bin/node

❯ nvm use
Now using Node v22.18.0 (npm 10.9.3) ~/.local/share/nvm/v22.18.0/bin/node

❯ node --version
v22.18.0
```